about.html

```diff
@@ -0,0 +1,250 @@
+      <h2 class="info-heading">Porsche Cars North America</h2>
+      <div class="info-underline"></div>
+      <p class="info-para">The Logo for our next model was decided through the logo competition from November 24 - to
+        December 11.
+      </p>
+
+      <p class="info-catchy-text">
+        PORSCHE AG the SUVs DEALERS.
+      </p>
+    </div>
+
+    <div class="hr-divider"></div>
+  </section>
+
+  <!-- INFORMATION SECTION STARTS -->
+
+  <!-- LATEST NEWS SECTION STARTS -->
+
+  <section class="latest-news">
+    <div class="latest-news-content">
+
+      <h2 class="latest-news-heading">See Latest 2022 Porsche Models</h2>
+
+      <div class="latest-news-underline"></div>
+
+      <p class="latest-news-para">Take a look at our latest 2022 Porsche Model making waves and driving celebrities
+        crazy.
+      </p>
+
+      <div class="overlay">
+        <div class="overlay-content">
+          <h2 class="overlay-title">Panamera</h2>
+          <p class="overlay-para">
+            The Porsche Panamera is, as far as anyone knows
+          </p>
+        </div>
+      </div>
+
+      <div class="overlay-two">
+        <div class="overlay-content-two">
+          <h2 class="overlay-title-two">Cayenne</h2>
+          <p class="overlay-para-two">
+            The Porsche Cayenne is, as far as anyone knows
+          </p>
+        </div>
+      </div>
+
+      <!-- OVERLAY FOR DESKTOP -->
+
+      <div class="overlay-desktop">
+        <div class="overlay-desk">
+          <div class="overlay-content-desk">
+            <h2 class="overlay-title-desk">Panamera</h2>
+            <p class="overlay-para-desk">
+              The Porsche Panamera is, as far as anyone knows
+            </p>
+          </div>
+        </div>
+
+        <div class="overlay-two-desk">
+          <div class="overlay-content-two-desk">
+            <h2 class="overlay-title-two-desk">Cayenne</h2>
+            <p class="overlay-para-two-desk">
+              The Porsche Cayenne is, as far as anyone knows
+            </p>
+          </div>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <!-- PARTNERS SECTION STARTS -->
+  <section class="partners" id="partners-desktop">
+
+    <h2 class="partners-heading">Our Partners</h2>
+    <div class="partners-underline"></div>
+
+    <div class="partners-cards">
+
+      <div class="partners-card">
+        <img src="images/partners/totalenergies.png" class="total-energy" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/netflix.png" class="netflix" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/coke.png" class="Coca Cola" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/facebook-1.png" class="Facebook" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/google-2.png" class="Google" />
+      </div>
+
+    </div>
+
+  </section>
```

Comment:
- [ ] In the **desktop** version of the about page, the partners section is hidden 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/203853551-a08b4ad1-b5dc-4b50-9c0f-31e2fc85c957.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/203853622-8d5234b2-50ae-40f3-a89b-1e30d6ae2f16.png)